ajax/libs/reactstrap/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "reactstrap",
+  "filename": "reactstrap.min.js",
+  "description": "React Bootstrap 4 components",
+  "homepage": "https://reactstrap.github.io/",
+  "keywords": [
+    "reactstrap",
+    "bootstrap",
+    "react",
+    "component",
+    "components",
+    "react-component",
+    "ui"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/reactstrap/reactstrap.git"
+  },
+  "license": "MIT",
+  "author": "Eddy Hernandez <edward.d.hernandez@gmail.com> (https://github.com/eddywashere)",
+  "npmName": "reactstrap",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "reactstrap.min.js",
+        "reactstrap.min.js.map"
```

Comment:
I think using `reactstrap.min.js*` is also OK, this way is more easy to read and flexible. 
If you can help us modify this two line to `reactstrap.min.js*`, it will be great.
But if you like to use rigorous, I think it's fine.